pkg/registry/apis/provisioning/jobs/migrate/mock_bulk_process_client.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	mock "github.com/stretchr/testify/mock"
 	metadata "google.golang.org/grpc/metadata"
-	
+
```

Comment:
This is some automatic fmt that I am not sure how to disable right now so will just leave it here 🤷